### ISSUE TITLE
MCP Pilot: Deploy Registry

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/planetary-data-cloud#35]
## 💡 Description

- https://github.com/NASA-PDS/registry/issues/222
- https://github.com/NASA-PDS/registry/issues/218
- https://github.com/NASA-PDS/registry/issues/223

**Comment by jordanpadams** [comment]
📆 May status: Not started

**Comment by jordanpadams** [comment]
:calendar: June status: Onboarding ongoing for some personnel. Completion will be delayed a couple weeks as onboarding took longer than expected.

**Comment by jordanpadams** [comment]
📆 July status: Onboarding ongoing for some personnel. Completion will be delayed a couple weeks as onboarding took longer than expected. No impact on system build.

**Comment by jordanpadams** [comment]
@jimmie has this been completed?

**Comment by jordanpadams** [comment]
:calendar: September status: Moving this to B13.1. No impact on build.

**Comment by jordanpadams** [comment]
📆 July status: Delayed start, but expected to be completed by delivery date. POC task so no impact on delivery.

**Comment by jordanpadams** [comment]
📆 August status: Deferred from B14.0. Working on more comprehensive policy definitions and terraform scripting. No impact on delivery to I&T.

**Comment by jordanpadams** [comment]
📆 September status: In work with MCP team to use serverless OpenSearch. Running into some issues with missing plugins, but marching forward. On schedule.

**Comment by jordanpadams** [comment]
📆 October status: Blocked by issues access OpenSearch and AWS technical team delays. Hopefully only delayed 1 sprint. In the meantime, working on https://github.com/NASA-PDS/planetary-data-cloud/issues/81

**Comment by jordanpadams** [comment]
📆 November status: Delayed 1 sprint due to delays working with AWS SMEs. Significant progress made and on schedule for completion by end of this sprint.

**Comment by jordanpadams** [comment]
All sub-tasks completed for time being. Will re-open new ticket when we need test/ops deployments
